### DISCUSSION TITLE
Create Upload Excel File Web Page

### DISCUSSION THREAD

**Discussion by img88** [qax-os/excelize#1668] [General]
Hi,

I'm a data analyst, however currently i got the project for creating web app in go, just for simple uploading excel file form. I try to find out a tutorial that give me full step by step how to create upload excel file form in Go. And i found that excelize is that i can possible to do it. However i'm still not find out a full tutorial how to create it. If you have any full tutorial i will be glad to see it.

Thank you

**Comment by xuri** [comment]
There are video tutorial in [the documentation websites](https://xuri.me/excelize/en/) for your reference.
